tests/js/spec/views/alerts/metricRules/metricField.spec.jsx

```diff
@@ -56,7 +56,20 @@ describe('MetricField', function () {
     openSelectMenu('(Required)');
 
     // 10 error aggregate configs
-    expect(screen.getAllByTestId('label')).toHaveLength(10);
+    [
+      'avg(…)',
+      'percentile(…)',
+      'p50(…)',
+      'p75(…)',
+      'p95(…)',
+      'p99(…)',
+      'p100(…)',
+      'failure_rate()',
+      'apdex(…)',
+      'count()',
+    ].forEach(label => {
+      expect(screen.getByText(label)).toBeInTheDocument();
+    });
```

Comment:
This is in the create alert page for metric alerts.
<img width="207" alt="Screen Shot 2022-06-10 at 2 19 59 PM" src="https://user-images.githubusercontent.com/44172267/173152800-fd4541c5-04df-48a1-9d6b-dca4c5565808.png">

Here we get the select options by test ids, which are added when we customize `SelectControl`'s `Option` component. Since that's no longer available, we can instead get options by their label texts. This is not as succinct, but it has the benefit of being more explicit about what we expect.